subprojects/docs/src/docs/userguide/ant.adoc

```diff
@@ -27,173 +27,271 @@ You can use Gradle's Ant integration as a path for migrating your build from Ant
+----
+
+.Output of `gradle hello`
+----
+> gradle hello
+include::{samplesPath}/userguide/ant/addBehaviourToAntTarget/addBehaviourToAntTarget.out[]
+----
 
 It is also possible for an Ant target to depend on a Gradle task:
 
-++++
-<sample id="dependsOnTask" dir="userguide/ant/dependsOnTask" title="Ant target that depends on Gradle task">
-            <sourcefile file="build.gradle"/>
-            <sourcefile file="build.xml"/>
-            <output args="hello"/>
-        </sample>
-++++
+=== Example: Ant target that depends on Gradle task
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/ant/dependsOnTask/build.gradle[]
+----
 
-Sometimes it may be necessary to “rename” the task generated for an Ant target to avoid a naming collision with existing Gradle tasks. To do this, use the api:org.gradle.api.AntBuilder#importBuild(java.lang.Object,org.gradle.api.Transformer)[] method.
+[source,xml]
+.build.xml
+----
+include::{samplesPath}/userguide/ant/dependsOnTask/build.xml[]
+----
 
-++++
-<sample id="renameAntDelegate" dir="userguide/ant/renameTask" title="Renaming imported Ant targets">
-            <sourcefile file="build.gradle"/>
-            <sourcefile file="build.xml"/>
-            <output args="a-hello"/>
-        </sample>
-++++
+.Output of `gradle hello`
+----
+> gradle hello
+include::{samplesPath}/userguide/ant/dependsOnTask/dependsOnTask.out[]
+----
 
-Note that while the second argument to this method should be a api:org.gradle.api.Transformer[], when programming in Groovy we can simply use a closure instead of an anonymous inner class (or similar) due to http://mrhaki.blogspot.ie/2013/11/groovy-goodness-implicit-closure.html[Groovy's support for automatically coercing closures to single-abstract-method types].
+Sometimes it may be necessary to “rename” the task generated for an Ant target to avoid a naming collision with existing Gradle tasks. To do this, use the link:{javadocPath}/org/gradle/api/AntBuilder.html#importBuild-java.lang.Object,%20org.gradle.api.Transformer-[AntBuilder.importBuild(java.lang.Object, org.gradle.api.Transformer)] method.
+
+=== Example: Renaming imported Ant targets
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/ant/renameTask/build.gradle[]
+----
+
+[source,xml]
+.build.xml
+----
+include::{samplesPath}/userguide/ant/renameTask/build.xml[]
+----
+
+.Output of `gradle a-hello`
+----
+> gradle a-hello
+include::{samplesPath}/userguide/ant/renameTask/renameAntDelegate.out[]
+----
+
+Note that while the second argument to this method should be a link:{groovyDslPath}/org.gradle.api.Transformer.html[Transformer], when programming in Groovy we can simply use a closure instead of an anonymous inner class (or similar) due to http://mrhaki.blogspot.ie/2013/11/groovy-goodness-implicit-closure.html[Groovy's support for automatically coercing closures to single-abstract-method types].
```

Comment:
The "Transformer" link looks wrong.  It's looking for it in the dsl ref, but it looks like it's only in the java/groovydocs.

Reply:
You're absolutely right. I'll fix these broken links.